<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>  Maschine Learning</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>
    <normalon>:/window-logo/ai-icon-9.png</normalon>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-color: #151819;}</string>
  </property>
  <property name="windowFilePath">
   <string>./pics</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="upperframe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color: #202124; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="lin_regression">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: bold; color: #C1C1C1; text-align: center; border: None}
QPushButton:hover{background-color: rgba(255,255,255,0.1);}</string>
     </property>
     <property name="text">
      <string>Regression</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="log_regression">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: bold; color: #C1C1C1; text-align: center; border: None}
QPushButton:hover{background-color: rgba(255,255,255,0.1);}</string>
     </property>
     <property name="text">
      <string>Classification</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="deep_learn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: bold; color: #C1C1C1; text-align: center; border: None}
QPushButton:hover{background-color: rgba(255,255,255,0.1);}</string>
     </property>
     <property name="text">
      <string>Deep Learning</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="FeaturListScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>92</y>
      <width>171</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea QWidget{   background-color:#282e30;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="DepFeaturFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>31</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color:#282e30; border: solid #C1C1C1; border-width: 0px 0px 1px 0px;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="DepFeaturLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {font: bold; color: #C1C1C1; text-align: center;  border-width: 0px 0px 0px 0px}</string>
     </property>
     <property name="text">
      <string>Dependent Featur:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="DepFeatComBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SetDepFeat">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{background-color: #202124;  color: #C1C1C1;}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_new_Project"/>
    <addaction name="actionOpen_Project"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionDark_Mode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionThere_is_no_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_new_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionDark_Mode">
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionThere_is_no_Help">
   <property name="text">
    <string>There is no Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/Qt-resourcses/logo.qrc"/>
 </resources>
 <connections/>
</ui>
